### PR TITLE
Feature/single recipe view2

### PR DESCRIPTION
**Pull request by aliroemhildt**
#### What's this PR do?
- added "-js" specific classes to all html elements that are being accessed in scripts.js
- completed functionality for: showing ingredients list, showing instructions

#### Where should the reviewer start?
- lines 54-108 of scripts.js

#### How should this be manually tested?
- open using the portal (run `npm start` and copy and past portal url)
- click on a recipe, the ingredients and instructions should be showing

#### Any background context you want to provide?
- the individual recipe page is still a WIP, I will complete the cost functionality later today

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
n/a

#### Questions:
n/a

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -22,8 +22,8 @@ <h1>What's Cookin'</h1>
     </div>
   </nav>
   <main>
-    <div class="flex-row main-view-container">
-      <section class="filter-section">
+    <div class="flex-row main-view-container main-view-container-js">
```
Comment (anewb87) [inline src/index.html:25]: LOVE this! So smart, so helpful.

**Review by cagallo — APPROVED** [review]
Great job, almost there!

File: src/scripts.js [hunk 2]
```diff
@@ -52,71 +52,74 @@ function displayAllRecipes() {
...
 function displaySelectedRecipe(e) {
-  if(e.target.classList.value !== 'favorite-button') {
-
+  if(e.target.classList.value !== 'favorite-button-js') {
     const image = document.querySelector('.selected-recipe-photo-js');
     const name = document.querySelector('.selected-recipe-name-js');
-    const ingredientList = document.querySelector('.selected-recipe-ingredient-list-js');
-    const cost = document.querySelector('.selected-recipe-cost-js');
-    const instructions = document.querySelector('.selected-recipe-instructions-js');
+    const costSection = document.querySelector('.cost-section-js');
```
Comment (anewb87) [inline src/scripts.js:58]: Appreciate the re-naming for empathy.

File: src/scripts.js [hunk 3]
```diff
@@ -52,71 +52,74 @@ function displayAllRecipes() {
...
+  return '<p class="selected-recipe-instructions">' + instructionsText + '</p>'
+};
 
 function show(elements) {
   for (var i = 0; i < elements.length; i++) {
     elements[i].classList.remove('hidden');
-    console.log('show')
-  }
-}
+  };
```
Comment (anewb87) [inline src/scripts.js:113]: Added semicolons for the win.

File: src/scripts.js [hunk 4]
```diff
@@ -52,71 +52,74 @@ function displayAllRecipes() {
...
-    //we have ingredient objects in an array (the quanity and unit)
-    //for each ingredient, we want to add a line to the innerText that says "ingredient name: quantity unit "
+  const ingredientNames = selectedRecipe.determineRecipeIngredients(ingredientsData);
 
-    // ingredientList.forEach(())
-    // cost.innerText =
-    // instructions.innerText =
-  }
-}
+  const ingredientListText = selectedRecipe.ingredients.reduce((acc, ingredient, index) => {
```
Comment (anewb87) [inline src/scripts.js:87]: Reduce! Coming in pretty darn handy.

First use of index in a reduce method.

File: src/scripts.js [hunk 5]
```diff
@@ -52,71 +52,74 @@ function displayAllRecipes() {
...
+    acc += `${ingredientNames[index]}: ${ingredient.quantity.amount}${ingredient.quantity.unit}<br>`;
+    return acc;
+  }, '');
+
+  return '<p class="selected-recipe-ingredients-list">' + ingredientListText + '</p>';
+};
+
+function getInstructionsElement(e, selectedRecipe) {
+  const instructionsSection = document.querySelector('.instructions-section-js');
```
Comment (anewb87) [inline src/scripts.js:96]: A non-global query selector!

File: src/scripts.js [hunk 6]
```diff
@@ -52,71 +52,74 @@ function displayAllRecipes() {
...
-}
+  const ingredientListText = selectedRecipe.ingredients.reduce((acc, ingredient, index) => {
+    acc += `${ingredientNames[index]}: ${ingredient.quantity.amount}${ingredient.quantity.unit}<br>`;
+    return acc;
+  }, '');
+
+  return '<p class="selected-recipe-ingredients-list">' + ingredientListText + '</p>';
+};
+
+function getInstructionsElement(e, selectedRecipe) {
```
Comment (anewb87) [inline src/scripts.js:95]: Great work. Excited for this.

**Review by anewb87 — APPROVED** [review]
Fantastic work. Excited about the progress.

**Pull request merged**